### PR TITLE
Naked Nest: You will become the Serpents Nest

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
Infective Parasitic abnormality that silently infects its victems and transforms them into additional nests. Also uses code that recalls the serpents back to their nest and reduces them to a nestedserpent charges on the nest.
Mostly proof of concept.
- Infected individual 0:00
image
-Infected Individual 3:00
image
-Individual becomes hostile 4:00
image
-2:40
image

Lobotomy Corp researchers didnt bother to see if there is a cure.

## Why It's Good For The Game
It asks managers if they are willing to harbor a extremely infective parasite for WAW level ego and a cannon that increases a creatures red damage coefficent by 0.2.

## Changelog
:cl:
add: Naked Nest
add: Skin Tone serpentgreen
add: Status Effect rendRedArmor & serpentshost
tweak: ice now cools you down past your core temp if you eat more than 25 units of it.
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Okay, Couple things
Just make the cures known in the records and we can merge.

**Comment by Kitsunemitsu** [comment]
I will be updating the maps to add in a wrench and ethanol to each one.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Okay, Couple things
Make the cures known in the records
When you get a good result you should get a cure vial to save and use
Add ethanol to the Nanomed vendors
Add a bit more time before conversion (make it 4 mins instead of 2)
and we can merge.

File: code/modules/mob/living/simple_animal/abnormality/waw/naked_nest.dm [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+		originalskintone = H.skin_tone
+	physical_symptoms = world.time + (180 SECONDS)
+
+/datum/status_effect/serpents_host/tick()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.adjustSanityLoss(-0.1) //the serpents final destination is your frontal lobe
+		H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.1)
+		if(H.bodytemperature <= INHOSPITABLE_FOR_NESTING || H.reagents.has_reagent(/datum/reagent/consumable/wellcheers_purple, 20) || H.reagents.has_reagent(/datum/reagent/consumable/red_queen, 20)) //cure conditions
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/waw/naked_nest.dm:281]: This is two wellcheers or a cup of inaccessible tea

Comment (InsightfulParasite) [reply code/modules/mob/living/simple_animal/abnormality/waw/naked_nest.dm]: Changed it to just temperature.

**Review by Kitsunemitsu — APPROVED** [review]

**Comment by Kitsunemitsu** [comment]
Has been tested many times, it works to a T. If there are any issues I'll wake para up to deal with them.

**Pull request merged**